service/src/main/kotlin/xyz/lebkuchenfm/external/storage/dropbox/DropboxClient.kt

```diff
@@ -117,11 +117,19 @@ class DropboxClient(config: ApplicationConfig) {
                 }
                 sendWithoutRequest { true }
                 refreshTokens {
-                    val tokenInfo: DropboxTokenInfo = client.post(tokenPostRequest).body()
-                    bearerTokenStorage.clear()
-                    bearerTokenStorage.add(
-                        BearerTokens(accessToken = tokenInfo.accessToken, refreshToken = refreshToken),
-                    )
+                    val tokenPostResponse = client.post(tokenPostRequest)
```

Comment:
This is fine for now, but I've noticed that `getInitialTokens` does mostly the same thing. Maybe we would like to make one method for getting the tokens.